src/index.html

```diff
@@ -10,8 +10,190 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="body">
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
```

Comment:
always write closed tag `>` on a new line 
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```